src/common/types.ts

```diff
@@ -1,15 +1,21 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-import { AwsCredentialIdentity, Provider, RequestSigner } from "@aws-sdk/types";
+import {
+  AwsCredentialIdentity,
+  AwsCredentialIdentityProvider as _AwsCredentialIdentityProvider,
+  RequestSigner,
+} from "@aws-sdk/types";
+
+export type AwsCredentialIdentityProvider = _AwsCredentialIdentityProvider;
```

Comment:
couldn't this be

`export { AwsCredentialIdentityProvider }` without the rename?

Reply:
Since `amazon-location-utilities-auth-helper-js` is going to depend on `aws-sdk` anyways, I think it is OK to just use `AwsCredentialIdentityProvider` from `@aws-sdk/types` instead of re-exporting.